src/pages/docs/ProvidingUISchemas.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Typography from "@material-ui/core/Typography";
+import withStyles from "@material-ui/core/styles/withStyles";
+import MarkdownElement from "../../common/MarkdownElement";
+import commonStyles from '../../common/styles';
+/* eslint import/no-webpack-loader-syntax: off */
+const registerUISchemaCode = require('!raw-loader!./listings/registerUISchema.md');
+const retrieveUISchemaCode = require('!raw-loader!./listings/retrieveUISchema.md');
+
+
+const styles = theme => ({
+  code: commonStyles.code(theme),
+  headline: commonStyles.headline,
+  title: {
+    marginTop: '0.5em'
+  },
+  display1: commonStyles.display1,
+  list: commonStyles.list,
+  link: commonStyles.link
+});
+
+export const ProvidingUISchemas = ({ classes }) => {
+
+  return (
+    <div>
+      <Typography variant={'display1'} className={classes.display1}>Providing UISchemas</Typography>
+
+      <p>
+        When initializing JSON Forms you have to provide a UISchema. In many cases this is enough as you can specify how the form should look like.
+        But in some cases especially when rendering arrays you must be able to provide a UISchema which can be retrieved.
+        This allows you to provide different UISchemas which will be selected by the renderer.
```

Comment:
Rephrease: This allows subsequenet renderers to use a different UI Schema than the the one specified initially